hack/istio/wait-for-namespace.sh

```diff
@@ -35,11 +35,14 @@ HELPMSG
   esac
 done
 
-for NAMESPACE in ${NAMESPACES[@]}; do
-  oc patch kiali kiali -n kiali-operator --type=json '-p=[{"op": "add", "path": "/spec/deployment/accessible_namespaces/0", "value":"'$NAMESPACE'"}]'
-done
+IS_MAISTRA=$([ "$(oc get crd | grep servicemesh | wc -l)" -gt "0" ] && echo "true" || echo "false")
```

Comment:
I think its safe to add MAISTRA related things into 1.73 since it will be last version for OSSM 2.x